Mathlib/Algebra/CharP/Invertible.lean

```diff
@@ -67,21 +71,21 @@ theorem CharP.isUnit_intCast_iff {z : ℤ} (hp : p.Prime) : IsUnit (z : R) ↔ 
 
 end Ring
 
-section Field
+section Semifield
 
-variable [Field K]
+variable [Semifield K]
```

Comment:
Should these changes be a separate PR? (For one, I would certainly like to benchmark them.)

Reply:
#25460